src/charm.py

```diff
@@ -65,21 +50,21 @@ def check_ranges(ranges, name):
 
     ranges is a string containing a comma-separated list of CIDRs, a CIDR being the only kind of valid element.
     """
-    networks = ranges.split(',')
+    networks = ranges.split(",")
     invalid_networks = []
     for network in networks:
         try:
             ip_network(network)
         except ValueError:
             invalid_networks.append(network)
     if invalid_networks:
-        return '{}: invalid network(s): {}'.format(name, ', '.join(invalid_networks))
+        return "{}: invalid network(s): {}".format(name, ", ".join(invalid_networks))
```

Comment:
I guess things like moving to f-strings is out of scope for this PR?

Reply:
This will be part of the rework as a sidecar